### DISCUSSION TITLE
LCZ converter - UMEP formatted land cover grid

### DISCUSSION THREAD

**Discussion by marciaamatias** [UMEP-dev/UMEP#607] [Q&A]
Hi everyone I need some help

I'm trying to run the SUEWS model by using the prepare window but I'm having trouble with the land cover grid we need to create. 

I've run the LCZ converter but the txt file It generates says it is all Paved. 

Is it possible to reclassify the raster with the LCZ into 1. Paved; 2. Buildings; and so on, and use it as a Land Cover Grid?
 
If this is not possible what is the solution? The land cover grid is mandatory to proceed to the next steps.

**Comment by biglimp** [comment]
Not sure I understand. In the LCZ converter, the zones are converted into the textfile used for SUEWS and each local climate zone will get fractions of grass, paved etc. distributed. You cannot convert LCZ data into landcover grid directly since they are valid for two different spatial scales.
